app/views/shared/vacancy/_jobseeker_view.html.slim

```diff
@@ -17,15 +17,15 @@
     .govuk-grid-row
       - if JobseekerApplicationsFeature.enabled? && @vacancy.apply_through_teaching_vacancies == "yes" && @vacancy.expires_at.future?
         .govuk-grid-column-one-third
-          - case job_application&.status
-          - when "submitted"
-            = render BannerButtonComponent.new text: t("jobseekers.job_applications.banner_links.submitted"),
-              href: jobseekers_job_application_path(job_application),
-              icon: "green-tick"
-          - when "draft"
-            = render BannerButtonComponent.new text: t("jobseekers.job_applications.banner_links.draft"),
-              href: jobseekers_job_application_path(job_application),
-              icon: "green-tick"
+          - if defined?(job_application) && ["withdrawn", nil].exclude?(job_application&.status)
+            - if job_application.status == "draft"
```

Comment:
If `job_application` is defined, then we can just do `if job_application.draft?`